doc/nrf/releases_and_maturity/migration/migration_guide_2.5.rst

```diff
@@ -0,0 +1,32 @@
+:orphan:
+
+.. _migration_2.5:
+
+Migration guide for |NCS| v2.5.0 (Working draft)
+################################################
+
+This document describes the changes required or recommended when migrating your application to |NCS| v2.5.0.
```

Comment:
to rephrase "to NCS 2.5.0 from NCS 2.4.0"

Reply:
This was originally phrased like this in [one of the previous commits](https://github.com/nrfconnect/sdk-nrf/commit/bc5fd287f9512f6aca5dd7274bd86061a285665e#diff-9a54356848c52708ddb8709157b048cf117565530c3b2a9ce982905e69b20768R8), rephrased to its current form after @carlescufi 's suggestion.

Reply:
I agree with @shanthanordic. I suggest we write:
```suggestion
This document describes the changes required or recommended when migrating your application from |NCS| v2.4.0 to |NCS| v2.5.0.
```

Reply:
Updated